vermouth/gmx/gro.py

```diff
@@ -107,6 +107,11 @@ def read_gro(file_name, exclude=('SOL',), ignh=False):
 
             molecule.add_node(idx, **properties)
             idx += 1
+        box = np.array(line.strip().split(), dtype=float)
+        # not pretty but the parser is out of touch with the rest of the parsing
+        # infrastructure already
+        molecule.box = box
```

Comment:
What's the problem with this?

Reply:
Ideally I'd not like to add "secret" class variables to molecule. Makes it a bit hard to know when this attribute is set and what possible values are. Having said that the only other option would be to put it into the meta dict. Both things are possible. Also box should ideally propagate to system but that's perhaps something for when we deal with coordinate building etc.

Reply:
well we can go either way; if you like it we can also just add it to molecule and put a doctoring with it